### PR TITLE
Reduce space taken up by settings

### PR DESCRIPTION
**Pull request by rschroll**
Several changes, mostly focused around reducing the size of the settings pane.  The big one is letting the server decide whether to show the (currently) Linux-only options.  I also adjusted some wordings, to try to clarify what they did.  There are a couple of fixes to minor bugs as well.

I'm happy to split these out into separate PRs, if you'd rather look at them individually.

### CODE REVIEW

File: www/static/style.css [hunk 1]
```diff
@@ -118,6 +121,12 @@ select {
 #displayoptions label {
     flex-grow: 1;
 }
+label input:disabled + span {
+    color: #888;
```
Comment (H-M-H) [inline www/static/style.css:125]: Does this also work for light mode? I guess two versions are required here.

File: www/templates/index.html [hunk 2]
```diff
@@ -22,29 +22,36 @@
             <div id="settings_scroll">
                 <h2>Settings</h2>
                 <section>
-                    <label for="window">Capture window:</label>
+                    <label for="window">Capture Window:</label>
```
Comment (H-M-H) [inline www/templates/index.html:25]: Should probably only be `Capture` instead of `Capture Window` as not only windows can be captured.

**Comment by H-M-H** [comment]
Thanks I like what you did here :) Apart from minor nitpicks this looks fine!

File: www/static/style.css (hunk 1 above)
Comment (rschroll) [reply www/static/style.css:125]: Actually, yes.  By choosing a value right in the middle, it's still readable in both modes while being obviously less prominent than non-disabled text.

That said, if you want to have this line repeated in both the modes, for ease of future development, that seems reasonable.

**Comment by H-M-H** [comment]
Thanks!

**Pull request merged**